libraries/freertos_plus/aws/ota/src/aws_iot_ota_agent.c

```diff
@@ -73,19 +73,19 @@ typedef struct OTAStateTableEntry
 } OTAStateTableEntry_t;
 
 /*
- * Returns the byte offset of the element 'e' in the typedef structure 't'.
- * Setting an arbitrarily large base of 0x10000 and masking off that base allows
- * us to do the same thing as a zero offset without the lint warnings of using a
- * null pointer. No structure is anywhere near 64K in size.
- */
+  * Returns the byte offset of the element 'e' in the typedef structure 't'.
+  * Setting an arbitrarily large base of 0x10000 and masking off that base allows
+  * us to do the same thing as a zero offset without the lint warnings of using a
+  * null pointer. No structure is anywhere near 64K in size.
+  */
 /*lint -emacro((923,9078),OFFSET_OF) Intentionally cast pointer to uint32_t because we are using it as an offset. */
 
 #define OFFSET_OF( t, e )    ( ( uint32_t ) ( &( ( t * ) 0x10000UL )->e ) & 0xffffUL )
```

Comment:
Why are we reinventing `offsetof()`?  It is [available in C89.](http://man7.org/linux/man-pages/man3/offsetof.3.html)

Reply:
Fixed in https://github.com/aws/amazon-freertos/pull/2025/commits/ec0eaa30008d145e3809864d7f6ce6f06f3b6f0a